[2017-06-26 10:07:26]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_BUSY]  The database file is locked (database is locked)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:822)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:171)

[2017-06-26 10:07:29]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_BUSY]  The database file is locked (database is locked)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:822)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:98)
	at handler.PortHandler.main(PortHandler.java:183)

[2017-06-26 10:08:22]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:171)

[2017-06-26 10:08:23]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:98)
	at handler.PortHandler.main(PortHandler.java:184)

[2017-06-26 10:09:04]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:171)

[2017-06-26 10:09:04]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:172)

[2017-06-26 10:09:04]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:173)

[2017-06-26 10:09:04]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:174)

[2017-06-26 10:09:04]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:98)
	at handler.PortHandler.main(PortHandler.java:184)

[2017-06-26 10:16:38]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:171)

[2017-06-26 10:16:38]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:172)

[2017-06-26 10:16:38]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:173)

[2017-06-26 15:10:21]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:171)

[2017-06-26 15:10:21]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:172)

[2017-06-26 15:10:21]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:173)

[2017-06-26 15:10:21]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:174)

[2017-06-26 15:10:21]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:98)
	at handler.PortHandler.main(PortHandler.java:184)

[2017-06-26 15:34:23]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:172)

[2017-06-26 15:34:23]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:173)

[2017-06-26 15:34:23]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:174)

[2017-06-26 15:34:57]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:171)

[2017-06-26 15:34:57]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:172)

[2017-06-26 15:34:57]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:173)

[2017-06-26 15:34:57]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:126)
	at handler.PortHandler.main(PortHandler.java:174)

[2017-06-26 15:34:57]-[ main ]-[ ERROR ] dao.DataDAO:62 >> org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_PRIMARYKEY]  A PRIMARY KEY constraint failed (UNIQUE constraint failed: data.id)
	at org.sqlite.core.DB.newSQLException(DB.java:909)
	at org.sqlite.core.DB.newSQLException(DB.java:921)
	at org.sqlite.core.DB.execute(DB.java:825)
	at org.sqlite.core.DB.executeUpdate(DB.java:863)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:99)
	at dao.DataDAO.insertData(DataDAO.java:60)
	at handler.PortHandler.save2DB(PortHandler.java:98)
	at handler.PortHandler.main(PortHandler.java:184)

[2017-06-26 19:56:21]-[ Timer-0 ]-[ ERROR ] timer.WriteExcel:87 >> 发送邮件失败
[2017-06-26 19:56:21]-[ Timer-0 ]-[ ERROR ] timer.WriteExcel:87 >> 发送邮件失败
[2017-06-26 19:56:21]-[ Timer-0 ]-[ ERROR ] timer.WriteExcel:88 >> com.sun.mail.smtp.SMTPSendFailedException: 554 DT:SPM 163 smtp2,DNGowADH4dxY9lBZth1eAw--.1695S2 1498478179,please see http://mail.163.com/help/help_spam_16.htm?ip=27.221.102.105&hostid=smtp2&time=1498478179

	at com.sun.mail.smtp.SMTPTransport.issueSendCommand(SMTPTransport.java:2267)
	at com.sun.mail.smtp.SMTPTransport.finishData(SMTPTransport.java:2045)
	at com.sun.mail.smtp.SMTPTransport.sendMessage(SMTPTransport.java:1260)
	at timer.SendAttachment.Send(SendAttachment.java:62)
	at timer.WriteExcel.writeExcel(WriteExcel.java:85)
	at timer.ReceiveMsgTask.run(ReceiveMsgTask.java:43)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)

[2017-06-26 19:56:21]-[ Timer-0 ]-[ ERROR ] timer.WriteExcel:88 >> com.sun.mail.smtp.SMTPSendFailedException: 554 DT:SPM 163 smtp2,DNGowADH4dxY9lBZth1eAw--.1695S2 1498478179,please see http://mail.163.com/help/help_spam_16.htm?ip=27.221.102.105&hostid=smtp2&time=1498478179

	at com.sun.mail.smtp.SMTPTransport.issueSendCommand(SMTPTransport.java:2267)
	at com.sun.mail.smtp.SMTPTransport.finishData(SMTPTransport.java:2045)
	at com.sun.mail.smtp.SMTPTransport.sendMessage(SMTPTransport.java:1260)
	at timer.SendAttachment.Send(SendAttachment.java:62)
	at timer.WriteExcel.writeExcel(WriteExcel.java:85)
	at timer.ReceiveMsgTask.run(ReceiveMsgTask.java:43)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)

